cmake/PrintUsefulCMakeInfo.cmake

```diff
@@ -1,14 +1,40 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
 
+# First append useful info to BuildInfoAndLibraryVersions.txt
+file(APPEND
+  "${CMAKE_BINARY_DIR}/BuildInfoAndLibraryVersions.txt"
```

Comment:
Just `BuildInfo.txt`? That's easier to tell people when they are supposed to send you the file :) and version info is also build info.